packages/auth/src/core/errors.ts

```diff
@@ -359,10 +365,22 @@ function _debugErrorMap(): ErrorMap<AuthErrorCode> {
       'different options. To avoid this error, call initializeAuth() with the ' +
       'same options as when it was originally called, or call getAuth() to return the' +
       ' already initialized instance.',
-    [AuthErrorCode.RECAPTCHA_CHECK_FAILED]:
-      'The ReCAPTCHA assessment failed for this request.',
+    [AuthErrorCode.MISSING_RECAPTCHA_TOKEN]:
+      'The reCAPTCHA token is missing when sending request to the backend.',
+    [AuthErrorCode.INVALID_RECAPTCHA_TOKEN]:
+      'The reCAPTCHA token is invalid when sending request to the backend.',
+    [AuthErrorCode.INVALID_RECAPTCHA_ACTION]:
+      'The reCAPTCHA action is invalid when sending request to the backend.',
     [AuthErrorCode.RECAPTCHA_NOT_ENABLED]:
-      'reCAPTCHA integration is not enabled for this project.'
+      'reCAPTCHA integration is not enabled for this project.',
+    [AuthErrorCode.MISSING_CLIENT_TYPE]:
+      'The reCAPTCHA client type is missing when sending request to the backend.',
+    [AuthErrorCode.MISSING_RECAPTCHA_VERSION]:
+      'The reCAPTCHA version is missing when sending request to the backend.',
+    [AuthErrorCode.INVALID_REQ_TYPE]:
+      'The reCAPTCHA client type or version is invalid when retrieving the site key.',
```

Comment:
These are the only error codes which are 1:1 mapped to the backend. While the acutal error will include the server error message which can be more generic. Is this what you're asking?

Reply:
I mean, could this error code "INVALID_REQ_TYPE" be used for other types of invalid requests (unrelated to recaptcha).. right now, looks like it is not. Asking since this is INVALID_REQ_TYPE and NOT INVALID_RECAPTCHA_REQ_TYPE.

Should we make the error string generic here - like "Invalid request parameters" ? If the backend sends a more detailed info, will the SDK pass it along?

Reply:
Hmm, I think it matches our error table. I think it's by design. It's not only used for recaptcha flow but can be extended later on. We can double check with backend though.